tests/phpunit/tests/blocks/block.php

```diff
@@ -567,4 +567,65 @@ public function test_build_query_vars_from_query_block_page_with_offset() {
 			)
 		);
 	}
+
+	/**
+	 * @ticket 53257
+	 */
+	public function test_block_has_support() {
+		$this->registry->register(
+			'core/example',
+			array(
+				'supports' => array(
+					'align'    => array( 'wide', 'full' ),
+					'fontSize' => true,
+					'color'    => array(
+						'link'     => true,
+						'gradient' => false,
+					),
+				),
+			)
+		);
+		$block_type    = $this->registry->get_registered( 'core/example' );
+		$align_support = block_has_support( $block_type, array( 'align' ) );
+		$this->assertSame( $align_support, true );
```

Comment:
There is `assertFalse` and `assertTrue` in PHPUnit :)

PHPUnit has also `@dataProvider` functionality that would be another alternative. It allows you to run the same test with different test data which in practice

Reply:
> assertFalse

That would make perfect sense and should have thought of that :) - made the changes.